qelem/lib/screens/splash_screen.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+class Splash extends StatefulWidget {
+  const Splash({Key? key, required Widget this.nextScreen}) : super(key: key);
+  final nextScreen;
+
+  @override
+  State<Splash> createState() => _SplashState();
+}
+
+class _SplashState extends State<Splash> {
+  void initState() {
+    super.initState();
+    _navigateToHome();
+  }
+
+  void _navigateToHome() async {
+    await Future.delayed(const Duration(seconds: 2), () {});
+    Navigator.pushReplacement(
+        context, MaterialPageRoute(builder: (context) => widget.nextScreen));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final productNameStyle = Theme.of(context).textTheme.titleLarge;
```

Comment:
its Headline 1 from primary text theme not titleLarge.
```suggestion
    final productNameStyle = Theme.of(context).primaryTextTheme.headline1;
```